ta/ta.mk

```diff
@@ -48,28 +48,45 @@ base-prefix := $(sm)-
 
 libname = utils
 libdir = lib/libutils
+ifeq ($(CFG_ULIBS_SHARED),y)
+libuuid = 71855bba-6055-4293-a63f-b0963a737360
+endif
 include mk/lib.mk
 
 CFG_TA_MBEDTLS_MPI ?= y
 ifeq ($(CFG_TA_MBEDTLS_MPI),y)
+mplib-for-utee = mbedtls
 $(call force,CFG_TA_MBEDTLS,y)
 else
+mplib-for-utee = mpa
 libname = mpa
 libdir = lib/libmpa
-include mk/lib.mk
+ifeq ($(CFG_ULIBS_SHARED),y)
+libuuid = 39b498d9-1e1f-4ae0-a9e1-6d1caf8ec731
```

Comment:
Wouldn't it make sense to have different uuids for mbedtls and mpa?

Reply:
Huh, yes, they are different already!?

Reply:
Sorry, my mistake.